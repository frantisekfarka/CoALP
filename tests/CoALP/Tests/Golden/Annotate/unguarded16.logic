q(s(s(G1,G2),G3),1):-q(s(G3,s(G1,G2)),2).
q(s(G3,s(G1,G2)),2):-q(s(s(G1,G2),G3),1).
