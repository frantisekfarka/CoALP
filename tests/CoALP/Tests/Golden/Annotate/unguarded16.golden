q(s(s(V_2, V_3), V_4), 1, transform-func-1(V_-8)) :- q(s(V_4, s(V_2, V_3)), 2, V_-8).
q(s(V_5, s(V_6, V_7)), 2, transform-func-2(V_-9)) :- q(s(s(V_6, V_7), V_5), 1, V_-9).
